irohad/maintenance/CMakeLists.txt

```diff
@@ -6,4 +6,5 @@
 add_library(maintenance metrics.cpp)
 target_link_libraries(maintenance
     prometheus-cpp::core prometheus-cpp::pull
+    async_subscription
```

Comment:
```suggestion
```
I suggest not to link libraries to subscriptions.

Reply:
okay, done

Reply:
done